src/main/kotlin/it/pagopa/wallet/services/WalletService.kt

```diff
@@ -108,6 +107,44 @@ class WalletService(
             }
     }
 
+    fun getWallet(walletId: UUID): Mono<WalletInfoDto> =
+        walletRepository
+            .findById(walletId.toString())
```

Comment:
why wallet GET endpoint accepts a UUID and not directly a string ?

Reply:
Because in the openapi descriptor wallet id is specified with UUID format to have strict type validation (and also requests validation either)